generated/nimodinst/nimodinst/session.py

```diff
@@ -146,11 +146,12 @@ def __init__(self, driver):
         self._item_count = 0
         self._current_item = 0
         self._interpreter = _library_interpreter.LibraryInterpreter('windows-1251')
-        # Note that _library_interpreter sets _handle to 0 in its constructor, so that if
+        # Note that _library_interpreter clears the session handle in its constructor, so that if
```

Comment:
NI-ModInst and NI-TClk have their own session.py.mako files. And since they don't yet support gRPC, using `_library_interpreter` is still fine, though it may be best to change it anyway for consistency.